_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,455 @@
+```
+
+#### Passed Example 11
+
+This paragraph includes the [visual reference word][] "right". The content in the second column is identified with the word "right" but also identified by referencing the word "howdy", thus matching the (**non-visual reference**) condition. Note that, despite the fact that the text is not visible, this example is applicable because it is included in the accessibility tree.
+
+```html
+<head>
+	<title>Passed example 11 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="/test-assets/non-visual-reference-alternative-9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p style="position:absolute; top:-9999em">Click the button labelled "howdy", on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 12
+
+This paragraph includes the [visual reference word][] "right". The content in the second column is identified with the word "right" but also identified by referencing the word "howdy", thus matching the (**non-visual reference**) condition. Note that, despite the fact that the text is not included in the accessibility tree, this example is applicable because it is visible.
+
+```html
+<head>
+	<title>Passed example 12 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="/test-assets/non-visual-reference-alternative-9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p aria-hidden="true">Click the green button labelled "howdy" for a surprise</p>
+		</div>
+		<div class="col">
+			<button style="background-color: green" onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 13
+
+This paragraph includes the [visual reference word][] "right". The user is told to find the navigation on the right and the navigation is also correctly identified by a `nav` element whose [accessible name][] contains the word "navigation", thus matching the (**non-visual reference**) condition.
+
+```html
+<head>
+	<title>Passed example 13 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="/test-assets/non-visual-reference-alternative-9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Passed Example 14
+
+No [text node][] on this page includes any of the [visual reference words][].
+
+```html
+<p>Click the button, for a surprise</p>
+<button onclick="alert('Surprise!')">Howdy</button>
+```
+
+#### Passed Example 15
```

Comment:
Not sure this test cases is necessary. Not a blocker though.

Reply:
That's the only example where the target does not include reference words.
OTOH, I agree that the non-visible part is misleading in that example. I'll try to improve it to focus on what it is supposed to show.

Reply:
Actually not… Moved it to Inapplicable example to illustrate the visibility/included in accessibility tree part.